packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
+
+**Best practices**
+
+- When the date picker is clicked, it takes today's date.
+- It is not possible to write the date from the keyboard.
+- It is not possible to change the date format.
```

Comment:
It's not possible to change the date format but the date format will be based on the browser locale. So it may differ if you use a french or an english setting

Reply:
That's very good to know! Thanks

Reply:
Would it be possible to activate the keyboard?

Reply:
With a bigger effort, yes

Reply:
Thanks for your answer Marvin. How bigger should the effort be? The current working in V3 enables only the date update through the keyboard, not the initial setup. Sorry for that but I think that this will be a requirement (at least for the time & datetime compos, hence my suggestion to make it possible for the date too) since we can't prevent the Content Manager users to select a time that would be different from 00, 15, 30, 45. Could we seize the "datetime" compo refacto slot to do that?

Reply:
Multiple days to make something consistent, working, accessible and tested I would say

Reply:
Thanks for your answer Marvin. We'll schedule those days then.